inc/sai.h

```diff
@@ -140,6 +142,8 @@ typedef enum _sai_api_t
     SAI_API_MY_MAC           = 45, /**< sai_my_mac_api_t */
     SAI_API_IPSEC            = 46, /**< sai_ipsec_api_t */
     SAI_API_GENERIC_PROGRAMMABLE = 47, /**<sai_generic_programmable_t */
+    SAI_API_ARS              = 48, /**< sai_ars_api_t */
+    SAI_API_ARS_QUALITY_MAP  = 49, /**< sai_ars_quality_map_api_t */
     SAI_API_MAX,                   /**< total number of APIs */
 } sai_api_t;
 
```

Comment:
In the previous commit there were additions to the Port Object (ARS enable and more). Removed from the commit ?

Reply:
Added it back. Thanks.
https://github.com/opencomputeproject/SAI/pull/1658/commits/4b7f7da34af48d86e3d72840256f19def1a369d2